omx_buffers/src/mfx_omx_srf_pool.cpp

```diff
@@ -254,6 +254,7 @@ mfxStatus MfxOmxSurfacesPool::FindAvailableSurface(OMX_BUFFERHEADERTYPE** ppBuff
 
     if (!bFound)
     {
+	MFX_OMX_LOG_ERROR(g_OmxLogLevel, "Failed to find output surface, maybe all buffers are allocated!");
```

Comment:
First, buffers allocated in another component, Media SDK OMX handles already allocated buffers. so comment not fully correct.
Function FindAvailableSurface(...) is called from UseBuffer(...) function (below), and tries to find available (non-locked) surface.

The main question from me - what exactly do you want to achieve by this change ?

Reply:
Upload new changes